### DISCUSSION TITLE
Few fixes for Finnish translation:

### DISCUSSION THREAD

**Discussion by quenthal** [Blackymas/NSPanel_HA_Blueprint#703] [General]
The string for climate 'on' is wrong as is please_confirm. Below are the correct ones:

 FIN:
      weekdays:
        mon: Maanantai
        tue: Tiistai
        wed: Keskiviikko
        thu: Torstai
        fri: Perjantai
        sat: Lauantai
        sun: Sunnuntai
      relative_day:
        today: Tänään
        tomorrow: Huomenna
        in_2_days: 2 päivän päästä
        in_3_days: 3 päivän päästä
        in_4_days: 4 päivän päästä
      climate:
        states:
          "on": _osoitteessa_ -> **on**
          "off": off
        heat: lämpö
      please_confirm: _Vahvistakaa_ -> **Vahvista**

**Comment by edwardtfn** [comment]
Thanks for that!
This was[ fixed in DEV](https://github.com/Blackymas/NSPanel_HA_Blueprint/commit/ebb37b85c893ef9e5cd8bfc6a089b3ab4226bb8e) and will be available in the next release.
